.env.sample

```diff
@@ -0,0 +1,4 @@
+FX_REMOTE_SETTINGS_WRITER_SERVER="https://settings-writer.prod.mozaws.net/v1"
```

Comment:
Why prefix with `FX_`?

Reply:
was following the pattern from https://github.com/mozilla/blurts-server/blob/c33a85b/scripts/updatebreaches.js which I was directed to from https://remote-settings.readthedocs.io/en/latest/support.html?highlight=automate#how-do-i-automate-the-publication-of-records-forever

Happy to drop the namespacing if it isn't bringing any value

Reply:
This script will run in isolation – it doesn't really share its environment with anything else, so we don't need any namespacing.

Reply:
Fine with me 👍 refactored to drop the `FX` namespace